src/test/kotlin/io/github/rybalkinsd/kohttp/ext/ResponseExtKtTest.kt

```diff
@@ -50,4 +51,63 @@ class ResponseExtKtTest {
             assertTrue { body!!.isNotEmpty() }
         }
     }
+
+    @Test
+    fun `gets response as string # ext`() {
+        val response = "https://postman-echo.com/get".httpGet().asString()
+        val expected = "{\"args\":{},\"headers\":{\"x-forwarded-proto\":\"https\",\"host\":\"postman-echo.com\",\"accept-encoding\":\"gzip\",\"user-agent\":\"okhttp/3.12.0\",\"x-forwarded-port\":\"443\"},\"url\":\"https://postman-echo.com/get\"}"
```

Comment:
it's possible to use multiline strings here and in other tests related to this PR

Reply:
I did try with multiline strings initially, but they don't indent well. 
Here's to add a little more context.

https://github.com/rybalkinsd/kohttp/pull/84/commits/ea2e6e7372d5a0bab472ca3a221a97c342678f6b#diff-3d2eecefb292c407d3ecd7e81486e1a9L84

Reply:
I'd suggest to have a look at [`trimMargin()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-margin.html)

Reply:
Good idea! Updated 👍